pennylane_lightning/core/src/simulators/lightning_tensor/cutn/base/CMakeLists.txt

```
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.20)
+
+project(lightning_tensor_tensornetBase LANGUAGES CXX)
```

Can we use `${PL_BACKEND}` throughout for clarity? 
```suggestion
project(${PL_BACKEND}_tensornetBase LANGUAGES CXX)
```